WebRoot/js/zimbraAdmin/domains/model/ZaDomain.js

```diff
@@ -741,6 +744,17 @@ function(tmpObj, newDomain) {
 		attr.setAttribute("n", ZaDomain.A_zimbraFeatureCalendarReminderDeviceEmailEnabled);
 	}
 
+	if(ZaDomain.extendedAttrs.length>0) {
+		for (var i = 0; i < ZaDomain.extendedAttrs.length; i++) {
+			for (var aname in tmpObj.attrs) {
+				if (ZaDomain.extendedAttrs[i] == aname) {
+					var attr = soapDoc.set("a", tmpObj.attrs[aname]);
+					attr.setAttribute("n", aname);
+				}
+			}
+		}
```

Comment:
Does this work?
```suggestion
		for (var i = 0; i < ZaDomain.extendedAttrs.length; i++) {
			var aname = ZaDomain.extendedAttrs[i];
			if (tmpObj.attrs[aname]) {
				var attr = soapDoc.set("a", tmpObj.attrs[aname]);
				attr.setAttribute("n", aname);
			}
		}
```

Reply:
Modified the code a bit more and it's working now.